virtcontainers/persist.go

```diff
@@ -224,6 +224,7 @@ func (s *Sandbox) dumpConfig(ss *persistapi.SandboxState) {
 		MachineAccelerators:     sconfig.HypervisorConfig.MachineAccelerators,
 		CPUFeatures:             sconfig.HypervisorConfig.CPUFeatures,
 		HypervisorPath:          sconfig.HypervisorConfig.HypervisorPath,
+		HypervisorPathList:      sconfig.HypervisorConfig.HypervisorPathList,
```

Comment:
I am yet to review the rest of the PR, so not sure how this is being used later.  But I do not understand why these variables are getting persisted at all. I would assume that these would just be needed at startup to validate the binary paths. And its not like they are reconfigured or changed at runtime.

Reply:
My experience was that I needed that for the mechanism to work in practice.

From reading the code, it looks like even container startup goes through `fetchSandbox`. I may be misunderstanding how this is supposed to work. Do you see any drawback to having the correct configuration for all other operations (besides a bit of extra memory)?

Reply:
Thank you for the explanation.